front/app/components/SignUpIn/AuthProviderButton.tsx

```diff
@@ -171,6 +184,14 @@ const AuthProviderButton = memo<Props>(
       []
     );
 
+    const isContinueEnabled = useCallback(() => {
```

Comment:
I believe `useCallback` returns a function. Maybe you would like to use `useMemo` or maybe just calculating the value every time without any memorization is fine too.